sensing/src/main/java/com/google/android/sensing/capture/CaptureViewModel.kt

```diff
@@ -239,9 +239,9 @@ class CaptureViewModel(application: Application) : AndroidViewModel(application)
    */
   fun invokeCaptureCompleteCallback() {
     CoroutineScope(context = Dispatchers.IO).launch {
-      SensingEngineProvider.getInstance(getApplication())
-        .onCaptureCompleteCallback(captureInfo)
-        .collect { captureResultLiveData.postValue(it) }
+      SensingEngine.getInstance(getApplication()).onCaptureCompleteCallback(captureInfo).collect {
```

Comment:
nit: Is the format right?

Reply:
Yes its correct. Just fitting in the column.